api/inputs/inputs.py

```diff
@@ -1,19 +1,32 @@
-class RegisterInputs():
+""" Validation Classes """
+
+
+class register_inputs():
+    """ register validation class"""
+
     def __init__(self):
         self.error = ''
-    def validate(self,inputs):
-        if 'names' in inputs and 'email' in inputs and 'password' in inputs and 'confirm_password' in inputs:
+
+    def validate(self, inputs):
+        """ Register validation method """
+        if ('names' in inputs and
+                'email' in inputs and
+                'password' in inputs and
+                'confirm_password' in inputs):
```

Comment:
Add validity checks for whether 'email' is an actual email, password is of correct length